dapper/mods/integration.py

```diff
@@ -12,8 +12,13 @@
 
 
 # fmt: off
-def rk4(f, x, t, dt, order=4):
-    """Runge-Kutta (explicit, non-adaptive) numerical ODE solvers.
+def rk4(f, x, t, dt, stages=4, s=0):
+    """Runge-Kutta (explicit, non-adaptive) numerical (S)ODE solvers.
+
+    The rule has strong / weak convergence order 1.0 for generic SDEs and order 4.0
+    convergence for ODEs when stages=4.  For stages=1, this becomes the Euler-Maruyama
+    schemefor SDEs (s > 0.0) with strong / weak convergence order 1.0 for SDEs with
+    additive noise as defined in the below. See `bib.grudzien2020numerical`.
```

Comment:
- Split deterministic and stochastic description into 2 paragraphs.
- Add reference for stochastic scheme.
- "Rule" --> "Scheme"

Reply:
Sounds good -- I'll take a whack at that in a bit.

Reply:
You'll need to see if you like the table for the orders of convergence, etc... see most recent commits.